src/rules/font_face.rs

```diff
@@ -257,75 +251,82 @@ impl<'i> ToCss for FontFormat<'i> {
   }
 }
 
-enum_property! {
-  /// A font feature tech descriptor in the `supports()`function of the
-  /// [src](https://drafts.csswg.org/css-fonts/#src-desc)
-  /// property of an `@font-face` rule.
-  pub enum FontFeatureTechnology {
-    /// Supports OpenType features.
-    OpenType,
-    /// Supports Apple Advanced Typography features.
-    AAT,
-    /// Supports Graphite features.
-    Graphite,
-  }
-}
-
-enum_property! {
-  /// A color font tech descriptor in the `supports()`function of the
-  /// [src](https://drafts.csswg.org/css-fonts/#src-desc)
-  /// property of an `@font-face` rule.
-  pub enum ColorFontTechnology {
-    /// Supports the `COLR` v0 table.
-    COLRv0,
-    /// Supports the `COLR` v1 table.
-    COLRv1,
-    /// Supports SVG glyphs.
-    SVG,
-    /// Supports the `sbix` table.
-    SBIX,
-    /// Supports the `CBDT` table.
-    CBDT,
-  }
-}
-
-/// A font technology descriptor in the `supports()`function of the
+/// A font format keyword in the `format()` function of the the
 /// [src](https://drafts.csswg.org/css-fonts/#src-desc)
 /// property of an `@font-face` rule.
 #[derive(Debug, Clone, PartialEq)]
 #[cfg_attr(
   feature = "serde",
   derive(serde::Serialize, serde::Deserialize),
-  serde(tag = "type", content = "value", rename_all = "kebab-case")
+  serde(tag = "type", content = "value")
 )]
 pub enum FontTechnology {
```

Comment:
You could probably use the `enum_property` macro to simplify this if you want. It would implement `Parse` and `ToCss` for you.